README.md

```diff
@@ -1,57 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Decode Morse
 
-> Description the project.
+> This project uses Ruby to decode morse code.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby
+- git
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+Clone the project by using terminal:
 
-### Setup
+```
+git clone https://github.com/mohamedelshamy55/Morse-code.git
 
-### Install
 
-### Usage
+ruby morse.rb
```

Comment:
- [ ] Here you have missed one step that takes me to the right folder where I can run the project. Include `cd Morse-code` after line 20.

Reply:
thanks for your comment